devfile.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
 components:
   - alias: typescript-plugin
     type: chePlugin
-    id: che-incubator/typescript/1.30.2
+    id: che-incubator/typescript/1.35.1
```

Comment:
I think you should use latest as only one version is kept by the plugin-registry

Reply:
Fair point.